include/realizations/config/routing.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef NGEN_REALIZATION_CONFIG_ROUTING_H
+#define NGEN_REALIZATION_CONFIG_ROUTING_H
+
+#include <boost/property_tree/ptree.hpp>
+
+#include "routing/Routing_Params.h"
+
+namespace realization{
+  namespace config{
+
+    static const std::string ROUTING_CONFIG_KEY = "t_route_config_file_with_path";
```

Comment:
More stylistic perhaps, but maybe this should be a `constexpr const char*`?
```suggestion
    static constexpr const char* ROUTING_CONFIG_KEY = "t_route_config_file_with_path";
```